src/mongocxx/test/spec/change_stream.cpp

```diff
@@ -91,7 +91,7 @@ void run_change_stream_tests_in_file(const std::string& test_path) {
             spec::apm_checker apm_checker;
             options::client client_opts;
             // "Begin monitoring all APM events for `client`"
-            apm_checker.skip_kill_cursors();
+            apm_checker.set_ignore_command_monitoring_event(apm_checker::event::kill_cursors);
```

Comment:
Since we'll skip more than "kill cursor" commands, I created an enum for different values. My original idea was to use strings, but a typo wouldn't be caught until the specified events aren't skipped, and the test fails.

Reply:
Good thinking, an enum does seem better.